source/game.debug.screen.page.playerbroadcasts.bmx

```diff
@@ -0,0 +1,914 @@
+			Local adString2c:String = a.GetSpotsSent() + "/" + a.GetSpotCount()
+			font.DrawBox( adString1a, x + adLeftX + 2, y+1 + entryPos*lineHeight*2 + lineHeight*0 + lineTextDY, adLeftX - 40, lineTextHeight, sALIGN_LEFT_CENTER, SColor8.White)
+			font.DrawBox( adString1b, x + adLeftX + 2 + adLineWidth-60-2, y+1 + entryPos*lineHeight*2 + lineHeight*0 + lineTextDY, 60, lineTextHeight, sALIGN_RIGHT_CENTER, secondLineCol)
+
+			font.DrawBox( adString2a, x + adLeftX + 2, y+1 + entryPos*lineHeight*2 + lineHeight*1 + lineTextDY, 60, lineTextHeight, sALIGN_LEFT_CENTER, secondLineCol)
+			font.DrawBox( adString2b, x + adLeftX + 2 + 65, y+1 + entryPos*lineHeight*2 + lineHeight*1 + lineTextDY, 55, lineTextHeight, sALIGN_CENTER_CENTER, secondLineCol)
+			font.DrawBox( adString2c, x + adLeftX + 2 + adLineWidth-55-2, y+1 + entryPos*lineHeight*2 + lineHeight*1 + lineTextDY, 55, lineTextHeight, sALIGN_RIGHT_CENTER, secondLineCol)
+
+			entryPos :+ 1
+		Next
+
+		Local countractCount:Int = entryPos
+		entryPos = 0
+		lineHeight = 12
+
+		For Local l:TProgrammeLicence = EachIn availableProgrammeLicences.Values() 'collection.GetProgrammeLicences()
+			If l.owner <> playerID Then Continue
+			'skip starting programme
+			If Not l.isControllable() Then Continue
+			'skip individual episodes
+			If l.isEpisode() Then Continue
+
+			Local oldAlpha:Float = GetAlpha()
+			If entryPos Mod 2 = 0
+				SetColor 0,0,0
+			Else
+				SetColor 60,60,60
+			EndIf
+			SetAlpha 0.75 * oldAlpha
+			DrawRect(x, y + entryPos * lineHeight, lineWidth, lineHeight-1)
+
+			Local changedTime:Int = GetChangedTime(l.GetID(), TVTBroadcastMaterialType.PROGRAMME)
+			If changedTime <> 0
+				SetColor 255,235,20
+				Local alphaValue:Float = 1.0 - Min(1.0, ((Time.GetTimeGone() - changedTime) / 5000.0))
+				SetAlpha Float(0.4 * Min(1.0, 2 * alphaValue^3))
+				SetBlend LIGHTBLEND
+				DrawRect(x, y + entryPos * lineHeight, lineWidth, lineHeight-1)
+				SetBlend ALPHABLEND
+			EndIf
+
+			'draw in topicality
+			SetColor 200,50,50
+			SetAlpha 0.65 * oldAlpha
+			DrawRect(x, y + entryPos * lineHeight + lineHeight-3, lineWidth * l.GetMaxTopicality(), 2)
+			SetColor 240,80,80
+			SetAlpha 0.85 * oldAlpha
+			DrawRect(x, y + entryPos * lineHeight + lineHeight-3, lineWidth * l.GetTopicality(), 2)
+
+			SetAlpha oldalpha
+			SetColor 255,255,255
+
+			Local progString:String = l.GetTitle()
+			font.DrawBox( progString, x+2, y+1 + entryPos*lineHeight + lineTextDY, lineWidth - 30, lineTextHeight, sALIGN_LEFT_CENTER, SColor8.White)
+
+			Local attString:String = ""
+'			local s:string = string(GetPlayer(playerID).aiData.Get("licenceAudienceValue_" + l.GetID()))
+			Local s:String = MathHelper.NumberToString(l.GetProgrammeTopicality() * l.GetQuality(), 4)
+			If s Then attString = "|color=180,180,180|A|/color|"+ s + " "
+
+			font.DrawBox(attString, x+2, y+1 + entryPos*lineHeight + lineTextDY, lineWidth-5, lineTextHeight, sALIGN_RIGHT_CENTER, SColor8.White)
+
+			entryPos :+ 1
+			If entryPos = 31
```

Comment:
Diesen Block hatte ich mit eingebaut, damit ab Film 32 die Liste rechts unter der Werbung beginnt. x wird nach rechts verrückt und y unter den Werbeblock gesetzt.
In Zeile 870 wird für den Koffer das y dann wieder ausschließlich entsprechend der Anzahl der Filme gesetzt. Dieses Problem ist aber nicht durch das Refactoring reingekommen.